azure-pipelines.yml

```diff
@@ -50,10 +50,10 @@ jobs:
           node_version: ^12.0.0
           webpack_version: latest
         node-10:
-          node_version: ^10.13.0
+          node_version: ^10.20.1
           webpack_version: latest
         node-10-canary:
-          node_version: ^10.13.0
+          node_version: ^10.20.1
```

Comment:
Why we update version?

Reply:
This is the latest version but you want to lock the v10.13.0?  Anyway, http2 is supported over v10.10.0 so I'm fine either way.

https://nodejs.org/api/http2.html

Reply:
@hiroppy azure and github actions by default install latest version if you use `^10.13`

Reply:
yes I know but I don't know why you are sticking to v10.13.0.

Reply:
@hiroppy by default we support minimum LTS

Reply:
No, LTS is just v10, not v10.13.